### PR TITLE
update cart and account page

### PR DESCRIPTION
**Pull request by anask601**

### CODE REVIEW

File: account.html [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+            <link rel="stylesheet" href="style.css">
+        </head>
+        <body>
+        <div class="container">
+            <div class="navbar">
+                <div class="logo">
+                    <a href="index.html"><img src="images/logo.png" width="125px"></a>
+                </div>
+                <nav>
+                    <ul id="MenuItems">
```
Comment (faran4engg) [inline account.html:19]: a side note: its better use lower case names for ids and classes

File: style.css [hunk 2]
```diff
@@ -303,11 +310,209 @@ select:focus{
     background: #ff523b;
     color: #fff;
 }
+/*------ single products details------ */
+.single-product{
+    margin-top: 80px;
+}
+.single-product .col-2 img{
```
Comment (faran4engg) [inline style.css:317]: it looks like prettier is not working on your VSCODE. I think you should enable Format On Save

**Review by faran4engg — APPROVED** [review]
minor comments. but approving the PR

File: account.html (hunk 1 above)
Comment (anask601) [reply account.html:19]: okay next time will remind this

File: style.css (hunk 2 above)
Comment (anask601) [reply style.css:317]: okay faran bhai

**Pull request merged**